eth/client.go

```diff
@@ -753,15 +754,16 @@ func (c *client) Vote(pollAddr ethcommon.Address, choiceID *big.Int) (*types.Tra
 func (c *client) Reward() (*types.Transaction, error) {
 	addr := c.accountManager.Account().Address
 
-	currentRound, err := c.CurrentRound()
+	tr, err := c.GetTranscoder(addr)
 	if err != nil {
 		return nil, err
 	}
 
-	ep, err := c.GetTranscoderEarningsPoolForRound(c.accountManager.Account().Address, currentRound)
+	ep, err := c.GetTranscoderEarningsPoolForRound(addr, tr.LastActiveStakeUpdateRound)
```

Comment:
Previously, we always fetched the earnings pool for the current round. When `LastActiveStakeUpdateRound` is in the past, the `TotalStake` field of the earnings pool for the current round would be 0 and the total stake of the orchestrator would be stored in the earnings pool for `LastActiveStakeUpdateRound`. This is resolved by always fetching the earnings pool for `LastActiveStakeUpdateRound` here.